adobe/react-spectrum
# fix(#3800): MobileSearchAutocomplete: focused style persists on blur

Closes #3800, Closes #3861

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue #3800](https://github.com/adobe/react-spectrum/issues/3800).

## 📝 Test Instructions:

1. Open https://reactspectrum.blob.core.windows.net/reactspectrum/e6d26f6dcd1b1755c95ac2cd19ceb083d123dbc1/storybook/index.html?path=/story/searchautocomplete--static
2. Open Developer Tools in web browser, and using the Responsive Design tools, select a breakpoint or device width that will render the SearchAutocomplete using the MobileSearchAutocomplete.
3. Verify that using Tab key to set focus to the MobileSearchAutocomplete renders the focus-ring style.
4. Pressing Tab again to blur the MobileSearchAutocomplete should remove the focus-ring style.

## 🧢 Your Project:
